lte/gateway/c/core/oai/test/amf/amf_app_test_util.cpp

```diff
@@ -164,6 +164,212 @@ int send_uplink_nas_registration_complete(
   return (rc);
 }
 
+/* Create pdu session establishment  request from ue */
+int send_uplink_nas_pdu_session_establishment_request(
+    amf_app_desc_t* amf_app_desc_p, amf_ue_ngap_id_t ue_id, const plmn_t& plmn,
+    const uint8_t* nas_msg, uint8_t nas_msg_length) {
+  bstring pdu_session_est_req;
+  tai_t originating_tai = {};
+
+  if ((!amf_app_desc_p) || (!nas_msg) || (0 == nas_msg_length)) {
```

Comment:
Please put variable on the left side of comparison

```suggestion
  if ((!amf_app_desc_p) || (!nas_msg) || (nas_msg_length == 0)) {
```

Reply:
addressed the comment